src/Ogooreck/API/ApiSpecification.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Net;
+using System.Net.Http.Json;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace Ogooreck.API;
+
+public static class ApiSpecification
```

Comment:
The class has no documentation comments, which would extremely useful for new users of your library

Reply:
This will come in a separate PR. I created a dedicated issue for it: https://github.com/oskardudycz/Ogooreck/issues/2.